src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyRowSetTest.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.sqlserver.jdbc.bulkCopy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import javax.sql.RowSetMetaData;
+import javax.sql.rowset.CachedRowSet;
+import javax.sql.rowset.RowSetFactory;
+import javax.sql.rowset.RowSetMetaDataImpl;
+import javax.sql.rowset.RowSetProvider;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentest4j.TestAbortedException;
+
+import com.microsoft.sqlserver.jdbc.RandomData;
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.SQLServerBulkCopy;
+import com.microsoft.sqlserver.jdbc.SQLServerStatement;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+public class BulkCopyRowSetTest extends AbstractTest {
+    
+    private static String tableName = AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("BulkCopyFloatTest"));
+    
+    @Test
+    public void testBulkCopyFloatRowSet() throws SQLException {
+        try (Connection con = DriverManager.getConnection(connectionString)) {
+            RowSetFactory rsf = RowSetProvider.newFactory();
+            CachedRowSet crs = rsf.createCachedRowSet();
+            RowSetMetaData rsmd = new RowSetMetaDataImpl();
+            rsmd.setColumnCount(1);
+            rsmd.setColumnName(1, "c1");
+            rsmd.setColumnType(1, java.sql.Types.FLOAT);
+            
+            crs.setMetaData(rsmd);
+            crs.setFloat(1, RandomData.generateReal(false));
+            
+            try (SQLServerBulkCopy bcOperation = new SQLServerBulkCopy(con)) {
+                bcOperation.setDestinationTableName(tableName);
+                bcOperation.writeToServer(crs);
+            }
+        }
+    }
+    
+    @BeforeEach

```

Comment:
I think you wanna run this method as `@BeforeAll` instead? As you could extend this table to include other datatypes in future and set this up once for all..

Reply:
It will probably stay as '@BeforeEach' in the future, so that the table is dropped and re-created before each test, separating each test run from other runs.

Reply:
Then how will u manage different datatypes - do u think you will need same table recreated for all tests?

Reply:
We might create the same table that contains all datatypes, we don't know what will be included yet. Whoever writes the test in the future can modify this line if they want.